player.c

```diff
@@ -157,7 +157,13 @@ void player_cmd(struct player *this, char *cmd)
 		cmd = "\033[C";
 	else if (*cmd == 'r')
 		cmd = "\033[D";
-	writestr(this->wpipe, cmd);
+	
```

Comment:
Remove trailing spaces.

Reply:
Actually, remove the blank line, and start the next with `else if`. This is part of the special commands group where we change them before forwarding to `omxplayer`.